public/js/custom.js

```diff
@@ -4,3 +4,15 @@ $(".like-button").on('click', function() {
     $(".like-button").text('LIKE COUNT: ' + JSON.parse(data).likes);
   });
 });
+
+$(".in-application-search input").on('keyup', function() {
+  var query = $(this).val();
+  $.get('/api/search/' + query, function(data) {
+    $(".search-results").html('');
```

Comment:
data is the HTTP Reponse body of the /api/search/ + query request. In other words it is the actual response of the server. JQuery code doesnt call the searchUsers function here our backend/server does that and returns a response.